docs/source/markdown/podman-create.1.md

```diff
@@ -563,6 +563,11 @@ Valid values are:
   **port_handler=rootlesskit**: Use rootlesskit for port forwarding.  Default.
   **port_handler=slirp4netns**: Use the slirp4netns port forwarding.
   **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2`).  Default to false.
+  **enable_ipv6=true|false**: Enable IPv6.  Default to false. (Required for `outbound_addr6`)
+  **outbound_addr=IPv4**: specify outbound ipv4 address slirp should bind to
+  **outbound_addr=INTEFACE**: specify outbound interface slirp should bind to (ipv4 traffic only)
+  **outbound_addr6=IPv6**: specify outbound ipv6 address slirp should bind to
+  **outbound_addr6=INTERFACE**: specify outbound interface slirp should bind to (ipv6 traffic only)
```

Comment:
It would be good to have this all in alphabetical order.

Reply:
done